Crash when launch app on Android Pie, because hasn't  foreground service permission

**Overview (Required)**

Cause `SecurityException`, launch on Android P devices.
reason, Hyperion use Foreground Service and Android P is required `FOREGROUND_SERVICE` permission.

```
Process: io.github.droidkaigi.confsched2019.debug, PID: 23887
java.lang.SecurityException: Permission Denial: startForeground from pid=23887, uid=10356 requires android.permission.FOREGROUND_SERVICE
    at android.os.Parcel.createException(Parcel.java:1950)
    at android.os.Parcel.readException(Parcel.java:1918)
    at android.os.Parcel.readException(Parcel.java:1868)
    at android.app.IActivityManager$Stub$Proxy.setServiceForeground(IActivityManager.java:5198)
    at android.app.Service.startForeground(Service.java:695)
    at com.willowtreeapps.hyperion.core.internal.HyperionService.attach(HyperionService.java:43)
    at com.willowtreeapps.hyperion.core.internal.HyperionService$Connection.onServiceConnected(HyperionService.java:154)
    at android.app.LoadedApk$ServiceDispatcher.doConnected(LoadedApk.java:1730)
    at android.app.LoadedApk$ServiceDispatcher$RunConnection.run(LoadedApk.java:1762)
    at android.os.Handler.handleCallback(Handler.java:873)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loop(Looper.java:193)
    at android.app.ActivityThread.main(ActivityThread.java:6718)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
 Caused by: android.os.RemoteException: Remote stack trace:
    at com.android.server.am.ActivityManagerService.enforcePermission(ActivityManagerService.java:9203)
    at com.android.server.am.ActiveServices.setServiceForegroundInnerLocked(ActiveServices.java:1189)
    at com.android.server.am.ActiveServices.setServiceForegroundLocked(ActiveServices.java:870)
    at com.android.server.am.ActivityManagerService.setServiceForeground(ActivityManagerService.java:20472)
    at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:976)
```


**Steps To Reproduce**

Build `debug` buildVeriants, and install app to Android P device.
(It seems that it doesn't reproduce on some devices. 🤔)


**Screenshots**

![image](https://user-images.githubusercontent.com/1962232/51159083-ae67a800-18ca-11e9-8689-13b56282784e.png)


**Env:**
 - Device: Pixel XL
 - OS: 9.1
 - `master` branch and `debug` buildVeriant

**Additional context**

This crash caused by Hyperion, and fixed after Hyperion version 0.9.25.
https://github.com/willowtreeapps/Hyperion-Android/issues/141

If want no updating Hyperion version, can fix it by writing into **debug AndroidManifest**.

```
<uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
```

--- a/buildSrc/src/main/java/dependencies/Dep.kt
+++ b/buildSrc/src/main/java/dependencies/Dep.kt
@@ -137,7 +137,7 @@ object Dep {
     }
 
     object Hyperion {
-        val version = "0.9.24"
+        val version = "0.9.25"
         val hyperionPlugins = listOf(
             "com.willowtreeapps.hyperion:hyperion-core:$version",
             "com.willowtreeapps.hyperion:hyperion-attr:$version",

